### PR TITLE
Update CGM page

### PR DESCRIPTION
**Pull request by MikePlante1**
* Ensure `/` are buffered by spaces
* Remove `xDrip` because it's not synonymous with xDrip4iOS
* Add ONE+ info to G7 paragraph
* Capitalization typo in header
* Correct `2+` to the official name `2 Plus`
* Remove `sensor` from list because `Sensor` is already declared in the header

### CODE REVIEW

File: docs/settings/devices/cgm.md [hunk 1]
```diff
@@ -3,27 +3,27 @@
...
 While using Nightscout as a CGM is an option, it should be avoided if possible because it will not keep Trio running in the background like other CGM options. You will have to open Trio manually to make it run loop cycles.
 
-## xDrip (xDrip4iOS)
+## xDrip4iOS
 To use xDrip4iOS as a CGM source, you must build it yourself with the same Apple Developer account you used to build your Trio app. You cannot use Shuggah or a version distributed by someone else's TestFlight. Please see the following for instructions on how to build xDrip4iOS yourself: [link](../../operate/build.md#xdrip4ios-as-cgm-source)
 
 However, if you are using Dexcom G6 or ONE with xDrip4iOS, you can choose the Dexcom G6 option in Trio rather than xDrip4iOS, and Trio will intercept the glucose readings even if you're using Shuggah or someone else's TestFlight of xDrip4iOS.
 
-## Dexcom G5/G6 (and ONE)
+## Dexcom G5 / G6 ONE
```
Comment (ebouchut) [inline docs/settings/devices/cgm.md]: I also suggest using `/` between `G6` and  `ONE` to be consistent.

**Review by ebouchut — COMMENTED** [review]
👍 LGTM.
I think you should add a slash which is missing.

File: docs/settings/devices/cgm.md (hunk 1 above)
Comment (MikePlante1) [reply docs/settings/devices/cgm.md]: Good catch. Thanks. Fix has been pushed.

**Review by tmhastings — APPROVED** [review]
LGTM

**Pull request merged**